guacamole/src/main/webapp/app/client/templates/client.html

```diff
@@ -96,6 +96,15 @@ <h3>{{'CLIENT.SECTION_HEADER_DEVICES' | translate}}</h3>
                     </div>
                 </div>
 
+                <!-- Connection parameters which may be modified while the connection is open -->
+                <div class="menu-section connection-parameters" id="connection-settings" ng-show="client.protocol">
+                    <guac-form namespace="getProtocolNamespace(client.protocol)"
+                               content="client.forms"
+                               model="menu.connectionParameters"
+                               model-only="true"
+                               data-disabled="true"></guac-form>
```

Comment:
Why is this disabled?

Reply:
I ... don't know. I will check, but I'm pretty sure it shouldn't be. This might be from testing.

Reply:
OK. Should be fixed now. That was from an earlier version of the interface where I was toying with having the form disabled by default and definitely shouldn't have still been there.